tests/validation/tests/v3_api/resource/terraform/longliving/aws/v1apps/main.tf

```diff
@@ -0,0 +1,213 @@
+  template_name     = "rancher-istio"
+  template_version  = "1.5.900"
+  target_namespace  = rancher2_namespace.istio.id
+  answers = {
+    "certmanager.enabled" = false
+    "enableCRDs" = true
+    "galley.enabled" = true
+    "gateways.enabled" = true
+    "gateways.istio-ingressgateway.resources.limits.cpu" = "2000m"
+    "gateways.istio-ingressgateway.resources.limits.memory" = "1024Mi"
+    "gateways.istio-ingressgateway.resources.requests.cpu" = "100m"
+    "gateways.istio-ingressgateway.resources.requests.memory" = "128Mi"
+    "gateways.istio-ingressgateway.type" = "NodePort"
+    "gateways.istio-ingressgateway.ports[0].nodePort" = 31380
+    "gateways.istio-ingressgateway.ports[0].port" = 80
+    "gateways.istio-ingressgateway.ports[0].targetPort" = 80
+    "gateways.istio-ingressgateway.ports[0].name" = "http2"
+    "global.monitoring.type" = "cluster-monitoring"
+    "global.rancher.clusterId" = rancher2_cluster_sync.custom-cluster.cluster_id
+    "istio_cni.enabled" = "false"
+    "istiocoredns.enabled" = "false"
+    "kiali.enabled" = "true"
+    "mixer.enabled" = "true"
+    "mixer.policy.enabled" = "true"
+    "mixer.policy.resources.limits.cpu" = "4800m"
+    "mixer.policy.resources.limits.memory" = "4096Mi"
+    "mixer.policy.resources.requests.cpu" = "1000m"
+    "mixer.policy.resources.requests.memory" = "1024Mi"
+    "mixer.telemetry.resources.limits.cpu" = "4800m",
+    "mixer.telemetry.resources.limits.memory" = "4096Mi"
+    "mixer.telemetry.resources.requests.cpu" = "1000m"
+    "mixer.telemetry.resources.requests.memory" = "1024Mi"
+    "mtls.enabled" = false
+    "nodeagent.enabled" = false
+    "pilot.enabled" = true
+    "pilot.resources.limits.cpu" = "1000m"
+    "pilot.resources.limits.memory" = "4096Mi"
+    "pilot.resources.requests.cpu" = "500m"
+    "pilot.resources.requests.memory" = "2048Mi"
+    "pilot.traceSampling" = "1"
+    "security.enabled" = true
+    "sidecarInjectorWebhook.enabled" = true
+    "tracing.enabled" = true
+    "tracing.jaeger.resources.limits.cpu" = "500m"
+    "tracing.jaeger.resources.limits.memory" = "1024Mi"
+    "tracing.jaeger.resources.requests.cpu" = "100m"
+    "tracing.jaeger.resources.requests.memory" = "100Mi"
+  }
+}
+
+resource "rancher2_cluster_logging" "logging" {
+  name = "cluster-logging"
+  cluster_id = rancher2_cluster_sync.custom-cluster.id
+  kind = "fluentd"
+  fluentd_config {
+    fluent_servers {
+        endpoint = var.logging_endpoint
+        weight = 100
+    }
+  }
+}
+
+// create aws resource & run docker run
+resource "aws_instance" "custom_nodes" {
+    count                   = length(local.node_config)
+    ebs_optimized           = true
+    instance_type           = "t3.xlarge"
+    ami                     = var.ami
+    subnet_id               = var.subnet
+    vpc_security_group_ids  = var.security_groups
+    key_name                = var.ssh_key_name
+    iam_instance_profile    = "RancherK8SUnrestrictedCloudProviderRoleUS"
```

Comment:
We should store `instance_type` and `iam_instance_profile` in vars for reusability and so that we don't reveal details about our internal aws infra